pulley/src/lib.rs

```diff
@@ -805,14 +805,30 @@ macro_rules! for_each_extended_op {
             fminimum32 = Fminimum32 { operands: BinaryOperands<FReg> };
             /// `low32(dst) = ieee_trunc(low32(src))`
             ftrunc32 = Ftrunc32 { dst: FReg, src: FReg };
+            /// `low128(dst) = ieee_trunc(low128(src))`
+            ftrunc32x4 = Ftrunc32x4 { dst: VReg, src: VReg };
```

Comment:
As a minor stylistic note, mind prefixing these instructions with `v` (e.g. `vtrunc32x4`) instead? I've been trying to use the `v` prefix for all simd/vector-related instructions so far

Reply:
yep